protocol/avdl/keybase1/user.avdl

```diff
@@ -142,6 +142,36 @@ protocol user {
 
   void uploadUserAvatar(string filename, union { null, ImageCropRect } crop);
 
+  ProfileProofSuggestionsRes profileProofSuggestions(int sessionID);
+  record ProfileProofSuggestionsRes{
+    array<ProfileProofSuggestion> suggestions;
+    boolean showMore;
+  }
+
+  // A proof the user doesn't have for the profile screen.
+  record ProfileProofSuggestion {
+    string key;
+    string text; // "Prove your Twitter", "Add a PGP key"
+    array<SizedImage> icon; // TODO CORE-9882: this is an empty list for now
+    array<Identify3RowMeta> metas; // for 'new' proof types
+  }
+
+  array<ProofSuggestion> proofSuggestions(int sessionID);
+
+  // A proof the user doesn't have for the "Prove your" picker.
+  record ProofSuggestion {
```

Comment:
why can't this be collapsed with `ProfileProofSuggestion`? it's nice to minimize the number of types so we can share more code for processing them..

Reply:
Good point. @chrisnojima do you care if `proofSuggestions` and `proofSuggestionsProfile` are merged?